src/jit/target.h

```diff
@@ -789,7 +789,7 @@ typedef unsigned short          regPairNoSmall; // arm: need 12 bits
 
   #define RBM_ALLINT              (RBM_INT_CALLEE_SAVED | RBM_INT_CALLEE_TRASH)
 #ifdef UNIX_AMD64_ABI
-  #define RBM_LOWINT              (RBM_EAX|RBM_RDI|RBM_RSI|RBM_EDX|RBM_ECX|RBM_EBX|RBM_ETW_FRAMED_EBP)
+  #define RBM_LOWINT              (RBM_EAX|RBM_ECX|RBM_EBX|RBM_ETW_FRAMED_EBP|RBM_R12|RBM_R13)
```

Comment:
I'm OK with this change. However, I would prefer to make the change to the LsraLimitSmallIntSet directly in lsra.h. I believe that I only used RBM_LOWINT because it happened to already be defined. I think it must have previously been used specifically to refer to the lower registers - but there doesn't appear to be any other reference to it. If you choose to make the change here, you should add a comment. Otherwise it will probably seem very strange to anyone reading the code.